packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer/MultiFrameViewerContainer.js

```diff
@@ -86,7 +95,15 @@ class MultiFrameViewerContainer extends React.Component {
   }
 
   onFrameChange(frame) {
-    const { setFrame } = this.props
+    const {
+      activeTool,
+      setFrame
+    } = this.props
+
+    if (activeTool?.marks?.size > 0) {
```

Comment:
```suggestion
    if (activeTool?.marks.size > 0) {
```

Marks are defined in the Tool model so you shouldn’t need to check for existence.

Reply:
I think per [new lines 32-34](https://github.com/zooniverse/front-end-monorepo/pull/2930/files#diff-ce0847accb83040e9a4dca672ec039b4ad28a27404b896384cd4aa8bcf861127R34) `activeTool` could be an empty object without a `marks` property, that was my reasoning.

Reply:
Oh good point. `activeTool` could be set to null if we aren't drawing, which would be caught by the first existence check but I don't know if that would break elsewhere. 🤷